examples/allocations_logement/code_construction_legislatif.catala_fr

```diff
@@ -264,7 +264,7 @@ n'est pas prise en compte dans le calcul de l'aide personnelle au logement.
 
 ```catala
 champ d'application ÉligibilitéAidePersonnelleLogement:
-  étiquette l822_5_2 exception r822_22_3 définition patrimoine_pris_en_compte
+  exception r822_22_3 définition patrimoine_pris_en_compte
```

Comment:
Thanks, but I'll have to review the logic of this code to make sure we're doing the right thing so I would prefer if this PR did not touch anything in `examples/allocations_logement`.

Reply:
Alright! I used this to check that the transformation was working on our real use-case, but was not sure about committing it either (and it was a good idea since at first the rewriting worked on trivial examples but not here)